force-app/main/default/objects/CQ_KR_SQX_Player__c/fields/CQ_KR_SQX_User__c.field-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>CQ_KR_SQX_User__c</fullName>
+    <deleteConstraint>SetNull</deleteConstraint>
+    <externalId>false</externalId>
+    <label>User</label>
+    <referenceTo>User</referenceTo>
+    <relationshipName>Players</relationshipName>
```

Comment:
we all follow naming conv in relationship name

Reply:
Ok sir